src/api/.haml-lint.yml

```diff
@@ -28,6 +28,7 @@ linters:
       - Metrics/BlockNesting
       - Metrics/LineLength
       - Naming/FileName
+      - Rails/OutputSafety
```

Comment:
@Ana06 if you know another way to shutup that cop, let me know

Reply:
I also had to "fight" with this lately, [this page](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Rails/OutputSafety) helped me win the fight!

Reply:
that worked